script.js

```diff
@@ -0,0 +1,384 @@
+console.log(getBillboardPrice('To be, or not to be', 20)); // 380
+
+// 9.Very simple, given a number (integer / decimal / both depending on the language), find its opposite (additive inverse).
+
+function opposite(number) {
+    return number - number * 2;
+}
+
+console.log(opposite(-0.75));
+
+// 10. In this simple assignment you are given a number and have to make it negative.
+// But maybe the number is already negative?
+
+function makeNegative(number) {
+    if (number >= 0) {
+        return number - number * 2;
+    } else {
+        return number;
+    }
+}
+
+console.log(makeNegative(5));
+
+// 11. It's pretty straightforward.
+// Your goal is to create a function that removes the first and last characters of a string.
+// You're given one parameter, the original string. You don't have to worry about strings with less than two characters.
+
+function removeCharacter(word) {
+    return word.slice(1,-1);
+}
+
+console.log(removeCharacter("precisely"));
+
+// 12. Your task is to create a function that does four basic mathematical operations.
+// The function should take three arguments - operation(string/char), value1(number), value2(number).
+// The function should return result of numbers after applying the chosen operation.
+
+function basicOp(operation, firstValue, secondValue) {
+    return eval(firstValue + operation + secondValue);
+}
+
+console.log(basicOp('*', 10, 2));
+
+// 13. The first century spans from the year 1 up to and including the year 100,
+// the second century - from the year 101 up to and including the year 200, etc.
+// Task
+// Given a year, return the century it is in.
+
+function century(year) {
+    return Math.ceil(year / 100);
+}
+
+console.log(century(1758));
+
+// 14. Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
+
+function evenOrOdd(number) {
+    if (number % 2 === 0) {
+        return 'Even';
+    } else  {
+        return 'Odd';
+    }
+}
+
+console.log(evenOrOdd(53));
+
+// 15. Create a function which answers the question "Are you playing banjo?".
+// If your name starts with the letter "R" or lower case "r", you are playing banjo!
+
+function areYouPlayingBanjo(name) {
+    const letterReturn = name.charAt(0);
+    if (letterReturn === 'R' || letterReturn === 'r') {
+        return name + ' plays banjo!';
+    } else {
+        return name + ' does not play banjo!';
+    }
+}
+
+console.log(areYouPlayingBanjo("Robert"));
+
+// 16. Write function bmi that calculates body mass index (bmi = weight / height2).
+//
+// if bmi <= 18.5 return "Underweight"
+// if bmi <= 25.0 return "Normal"
+// if bmi <= 30.0 return "Overweight"
+// if bmi > 30 return "Obese"
+
+function BMI(weight, height) {
+    const bmiValue = weight / (height * height);
+    if (bmiValue <= 18.5) {
+        return 'Underweight';
+    } else if (bmiValue <= 25) {
+        return 'Normal';
+    } else if (bmiValue <= 30) {
```

Comment:
This `else` statement is not necessary

Reply:
I don't undesrstand how can i fixed it.

Reply:
https://doc.clickup.com/24538860/d/h/qcvqc-4212/46fac16fbd304d9
![Screenshot from 2024-04-01 15-47-02](https://github.com/mikolajkus/Function/assets/15181215/c3b904d6-ddb8-41e0-8488-2b64f897a086)